cpp/src/io/parquet/writer_impl.cu

```diff
@@ -851,6 +850,10 @@ void writer::impl::write(table_view const &table)
     // increment num rows
     md.num_rows += num_rows;
   }
+  // Create table_device_view so that corresponding column_device_view data
+  // can be written into col_desc members
+  auto parent_column_table_device_view = table_device_view::create(table);
```

Comment:
Should this also be a uvector? CC @devavret 
Edit: I think the name verbosity suggests that this should be a vector, as it then becomes simply parent_column_views.

Reply:
It's creating the table device view using the input table. We need the generated column device views.